Makefile

```diff
@@ -14,38 +14,23 @@
 # `clean` run clean on esptool2, rboot, and rboot-sampleproject with inherited
 #				 configuration
 
-ESPTOOL2      ?= $(abspath esptool2/esptool2.exe)

```

Comment:
i'd still export this, if someone wants to override it they can do it here instead of in each of the make files below, same with the sdk_base